inc/views/template_parts.php

```diff
@@ -107,6 +108,15 @@ public function render_featured_post() {
 					'post_status' => 'publish',
 				)
 			);
+
+			/**
+			 * Filters the featured posts.
+			 *
+			 * @since 3.5.0
+			 *
+			 * @param array $posts Array of posts.
+			 */
+			$posts = apply_filters( 'neve_filter_featured_posts', $posts );
```

Comment:
I think we should update the `param` tag description/typehint value and explain that it can also accept post types `int[]` beside the post objects.